orb-slam3/README.md

```diff
@@ -0,0 +1,35 @@
+# ORB-SLAM3 Docker image
+This repo contains ready Dockerfile for building ORB-SLAM3 image. You can use original version or modification for working with two cameras simultaneously.
```

Comment:
It supports original ORB SLAM version as well as our modification with 2 RGB-D cameras support